### PR TITLE
Start a11y testing checklist in Forem docs

### PR DESCRIPTION
**Pull request by marcysutton**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update

## Description

An accessibility section was recently [introduced to the PR template](https://github.com/forem/forem/pull/11407). To assist testing through development and PR reviews, this change includes a checklist of suggested accessibility testing steps in the Forem docs. It's meant to be a gentle guide that doesn't overwhelm reviewers who may be new to accessibility–feedback welcome on whether it hits the mark for that purpose. There's always more detail and nuance that could be added, so I tried to strike a balance with _just enough information_ to have a positive impact.

## Related Tickets & Documents

PR template PR 🤯 https://github.com/forem/forem/pull/11407

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how accessibility is impacted and tested. For more info, check out the [Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

^^ Isn't this section awesome?? 😍

## Added tests?

- [ ] Yes
- [X] No, and this is why: Not applicable to docs changes
- [ ] I need help with writing tests

## Added to documentation?

- [X] [Developer Docs](https://docs.forem.com) and/or [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [ ] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

otter grooving with a fish

### CODE REVIEW

File: docs/tests/accessibility-tests.md [hunk 1]
```diff
@@ -34,3 +35,56 @@ in your unit tests with [jest-axe](https://github.com/nickcolley/jest-axe).
...
+
+Pull Requests should include accessibility testing to prevent barriers from
+making it into the codebase. Sometimes issues found may be out of scope for a
+particular PR – those can be opened as separate issues to be followed up on.
+
+Test the functionality in web browsers for accessibility impact: Chrome,
+Firefox, and IE 11 were popular for Windows screen reader users in 2019
+[according to WebAIM](https://webaim.org/projects/screenreadersurvey8/#browsers).
+Safari is the most common choice for Voiceover users on the Mac, even if Chrome
+is widely used for development. Mobile browsers are worth checking, too.
```
Comment (rhymes) [inline docs/tests/accessibility-tests.md:49]: FYI: we don't support IE11. AFAIK the website doesn't work anymore in IE11, only Edge versions based on Chromium. 

cc @nickytonline

**Review by rhymes — COMMENTED** [review]
Hi @marcysutton, love the checklist! Just left a note about mentioning IE11

File: docs/tests/accessibility-tests.md (hunk 1 above)
Comment (nickytonline) [reply docs/tests/accessibility-tests.md:49]: > FYI: we don't support IE11. AFAIK the website doesn't work anymore in IE11, only Edge versions based on Chromium. 
> 
> 
> 
> cc @nickytonline 

Correct @rhymes.

Comment (marcysutton) [reply docs/tests/accessibility-tests.md:49]: got it! We can remove that part.

**Review by jacobherrington — APPROVED** [review]
:heart_eyes: This is great! Thanks @marcysutton!

**Review by benhalpern — APPROVED** [review]

**Review by rhymes — APPROVED** [review]

**Review by nickytonline — APPROVED** [review]
Thanks so much for this @marcysutton! 🔥 🚀

**Pull request merged**